Dungeoneer/src/com/interrupt/dungeoneer/net/DownloadListener.java

```diff
@@ -0,0 +1,11 @@
+package com.interrupt.dungeoneer.net;
+
+import com.badlogic.gdx.files.FileHandle;
+
+public interface DownloadListener {
```

Comment:
It might be nice to have a progress callback? For the future UI it would be nice to have some indicator of how much download is left.
```java
void progress(DownloadProgress progress);
```

Reply:
Yes, totally agree.

Reply:
Implemented!